docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,289 @@
+# Proposal A
+
+This proposal defines a new artifact manifest and corresponding referrers extension API for registries to support reference types.
+
+The modifications include:
+
+- [New artifact manifest](#artifact-manifest)
+- [Descriptor (OPTIONAL) additional property](#descriptor-properties)
+- [Distribution - Referrers API](#registry-http-api)
```

Comment:
Should be short description which fits in the README table

Reply:
Could you just lift line 3? 

I can remove line 5, and the bullets below. I was just trying to make it easier to find the related changes as this proposal is more detailed.